### PR TITLE
Fixes #10666 - rename alias config files

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: snippets/kickstart_networking_setup.erb [hunk 1]
```diff
@@ -72,7 +72,14 @@ real=`ip -o link | grep <%= interface.respond_to?(:inheriting_mac) ? interface.i
 real=`echo <%= interface.identifier -%> | sed s/<%= interface.attached_to -%>/$real/`
 <% end -%>
 
-cat << EOF > /etc/sysconfig/network-scripts/ifcfg-$real
+# ifcfg files are ignored by NM if their name contains colons so we convert colons to underscore
+<% if alias_type %>
```
Comment (ekohl) [inline snippets/kickstart_networking_setup.erb]: I think you want ```-%>``` here (and all other lines you change)

**Comment by ares** [comment]
right, thanks

**Comment by ekohl** [comment]
:+1:

**Comment by domcleal** [comment]
Thanks @ares.

**Pull request merged**